examples/vision/cutmix.py

```diff
@@ -0,0 +1,391 @@
+"""
+Title: CutMix data augmentation for image classification<br>
+Author: [Sayan Nath](https://twitter.com/SayanNa20204009)<br>
+Date created: 2021/04/14<br>
+Last modified: 2021/04/14<br>
+Description: Data augmentation with CutMix for image classification on CIFAR-10.
+"""
+
+"""
+## Introduction
+"""
+
+"""
+_CutMix_ is a data augmentation technique that addresses the issue of
+information loss and inefficiency present in regional dropout strategies. 
+Instead of removing pixels and filling them with black or grey pixels or
+Gaussian noise, you replace the removed regions with a patch from another image,
+while the ground truth labels are mixed proportionally to the number of pixels
+of combined images. The CutMix technique can make efficient use of training
+pixels, while retaining the regularization effect of regional dropout.
+This data augmentation technique was proposed in
+[CutMix: Regularization Strategy to Train Strong Classifiers with Localizable
+Features](https://arxiv.org/pdf/1905.04899.pdf)
```

Comment:
- Keeping the link on one line (as per @fchollet 's suggestion)
- Avoiding repetition ("technique... technique" -> "technique... method")

```suggestion
of combined images. The CutMix technique can make efficient use of training
pixels, while retaining the regularization effect of regional dropout.
This data augmentation method was proposed in
[CutMix: Regularization Strategy to Train Strong Classifiers with Localizable Features](https://arxiv.org/pdf/1905.04899.pdf)
```